src/price_oracle.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package helium;
+
+message price_oracle_resp_v1 {
+  /// The price
+  uint64 price = 1;
```

Comment:
indicate units

Reply:
That would be bones still correct? 1 HNT = 10^8 bones?

Reply:
Should it be 10^6 now/going forward to start the transition since we know that’s a limitation on solana?

Reply:
I've updated the comment to reflect 1$ = 10^6